components/patterns/lockup.tsx

```diff
@@ -1,19 +1,24 @@
-import {DrupalLink} from "@/components/simple/link";
+import Link from "next/link";
 
-export const Lockup = () => {
+import Logo from "@/components/simple/logo";
+
+export const Lockup = (props) => {
   return (
-    <div>
-      <DrupalLink href="/" className="su-cc su-lockup su-no-underline su-inline-block">
-        <div className="su-flex su-flex-col md:su-flex-row su-basefont-19">
-          <div
-            className=" su-logo su-text-cardinal-red su-type-4 su-leading-half su-pt-11 su-pr-7 su-mr-7 md:su-border-r su-border-solid su-border-black "> Stanford
-          </div>
-          <div
-            className=" su-text-25 md:su-text-m2 su--mt-3 md:su-mt-0 su--ml-2 md:su-ml-0 su-font-regular su-relative su-top-6 su-text-black ">
-            {process.env.NEXT_PUBLIC_SITE_NAME}
+    <div {...props}>
+      <Link href="/" className="su-no-underline">
+        <div className="su-flex md:su-flex-row su-basefont-19">
+          <Logo/>
+          <div className="lg:su-flex">
+            <div
+              className="lg:su-h-[45px] su-logo su-text-cardinal-red su-type-4 su-leading-[initial] su-pr-5 su-mr-5 lg:su-border-r su-border-solid su-border-black">
```

Comment:
Looks like the recent [decanter font addition](https://github.com/SU-SWS/sulgryphon-nextjs/pull/24/files#diff-4ee0e71a2145586038d583d58bcbd7aaa6b42318f417f5b62f2cbee48931861b) are causing some adjustment issues that will need to be fixed. And can we add some more spacing between the logo and `Stanford`?
<img width="240" alt="Screen Shot 2022-11-21 at 12 01 53 PM" src="https://user-images.githubusercontent.com/49299083/203147922-693aa036-c83d-49af-8c3c-16deb475077e.png">